### PR TITLE
Hpc upgrade docs

### PR DESCRIPTION
**Pull request by jmvera255**
#91

PR also now includes two additional files for an HPC-specific layout template that (if the code is right) will create linked buttons at the top of the page just like the "General Computing", "HTC System", "HPC Cluster", etc. buttons at [All User Guides](http://chtc.cs.wisc.edu/guides.shtml) 

Also still need to decide how/where else we want to include links in the text to other HPC guides

### CODE REVIEW

**Comment by jmvera255** [comment]
@tjslauson will there be scratch space? I'll want to include that info accordingly

**Comment by tjslauson** [comment]
@jmvera255

|         | Local scratch?                | Shared scratch?     |
| ------- | ----------------------------- | ------------------- |
| Login   | No                            | `/home/<username>`  |
| Execute | `/scratch/local/<username>`   | `/home/<username>`  |

**Comment by tjslauson** [comment]
@jmvera255 we talked more about this today and decided to add local scratch to login nodes.  They'll have both local and shared scratch space with the same paths as execute nodes.

File: hpc-overview-el7.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+the `univ`, `univ2`, `pre`, and `int` partitions which are available to 
+all HPC users as well as research group specific partitions that consist 
+of researcher owned hardware and which all HPC users can access on a 
+backfill capacity via the `pre` partition (more details below).
+
+### Partitions
+
+{:.gtable}
+  | Partition | p-name | \# nodes (N) | t-max | t-default | max nodes/job | cores/node (n) | RAM/node (GB) |
+  | --- |
```
[the remark this answers is not in the document]
Comment (jmvera255) [reply hpc-overview-el7.md:109]: Do you mean like adding a row for each "private" partition? I would be in favor of adding a row to the table that cumulatively details the hardware available across researcher owned partitions (which I understand is not that straightforward to do and would likely include a range of hardware configurations)

**Pull request merged**